src/main/java/store/view/OutputView.java

```diff
@@ -0,0 +1,75 @@
+package store.view;
+
+import store.model.GiftProduct;
+import store.model.Product;
+import store.model.receipt.AmountInfo;
+import store.model.receipt.GiftProducts;
+import store.model.receipt.PurchaseProducts;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class OutputView {
+    public void printStoreMenu() throws IOException {
+        System.out.println("""
+                안녕하세요. W편의점입니다.
+                현재 보유하고 있는 상품입니다.
+                """);
+
+        BufferedReader reader = new BufferedReader(new FileReader("src/main/resources/products.md"));
+        String product;
+
+        reader.readLine();
+
+        while ((product = reader.readLine()) != null) {
+            String[] str = product.split(",");
+            System.out.printf("- %s %s원 %s %s", str[0], String.format("%,d", Integer.parseInt(str[1])), changeNoStock(str[2]), changeNull(str[3]));
+            System.out.println();
+        }
+        reader.close();
+    }
+
+    public void printPurchaseProduct(PurchaseProducts purchaseProducts) {
+        System.out.println("==============W 편의점================");
+        System.out.println("상품명\t\t\t\t수량\t\t금액");
+        for (Product product : purchaseProducts.getPurchaseProducts()) {
+            System.out.printf("%-4s\t\t\t\t%s\t\t%s\n", product.getName(), product.getQuantity(), String.format("%,d", product.getQuantity() * product.getPrice()));
+        }
+    }
+
+    public void printGiftProducts(GiftProducts giftProducts) {
+        System.out.println("=============증\t\t정===============");
+        for (GiftProduct giftProduct : giftProducts.getGiftProducts()) {
+            System.out.printf("%-4s\t\t\t\t%d", giftProduct.getName(), giftProduct.getQuantity());
+            System.out.println();
+        }
+    }
+
+    public void printAmountInfo(AmountInfo amountInfo) {
+        System.out.println("====================================");
+        System.out.printf("총구매액\t\t\t\t%s\t\t%s", amountInfo.getTotalPurchaseCount(), String.format("%,d", amountInfo.getTotalPurchaseAmount()));
+        System.out.println();
+        System.out.printf("행사할인\t\t\t\t\t\t%s", String.format("-%,d", amountInfo.getPromotionDiscount()));
```

Comment:
```suggestion
        System.out.printf("행사할인\t\t\t\t\t\t-%,d", amountInfo.getPromotionDiscount());
```
이렇게 사용하면 ``System.out.printf()``와 ``String.format()``를 함께 사용하지 않고도 출력할 수 있지 않을까요?